msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -5,42 +5,47 @@ description: Identify the meeting apps API references with examples and Code sam
-|**CART**|Enables you to post captions for a meeting, which was started.|**POST /cartcaption?meetingid=04751eac-30e6-47d9-9c3f-0b4ebe8e30d9&token=04751eac&lang=en-us HTTP/1.1**|Microsoft Teams client SDK|
+|Method| Description| Source|
+|---|---|----|
+|[**Get user context**](#get-user-context-api)| Get contextual information to display relevant content in a Teams tab.| MSTC SDK|
+|[**Get participant**](#get-participant-api)| Fetch participant information by meeting ID and participant ID. |MSBF SDK|
+|[**Send notification signal**](#send-notification-signal-api)| Provide meeting signals using the existing conversation notification API for user-bot chat and allows to notify user action that shows an in-meeting dialog box. |MSBF SDK|
+|[**Get meeting details**](#get-meeting-details-api)| Get a meeting's static metadata. |Bot SDK |
+|[**Send CART API**](#send-cart-api)|Enable you to post captions for a meeting, which was started. |MSTC SDK|
+|[**Share app content to stage**](#share-app-content-to-stage-api)| Share specific parts of the app to meeting stage from the app side panel in a meeting. |MSTC SDK|
+|[**Get app content stage sharing state**](#get-app-content-stage-sharing-state-api)| Fetch information about apps' sharing state on the meeting stage. |MSTC SDK|
+|[**Get app content stage sharing capabilities**](#get-app-content-stage-sharing-capabilities-api)| Fetch the apps' capabilities for sharing to the meeting stage. |MSTC SDK|
+|[**Get real-time Teams meeting events**](#get-real-time-teams-meeting-events-api)|Fetch real-time meeting events, such as actual start and end time.| Bot SDK|
 
-## GetUserContext API
+## Get user context API
 
-To identify and retrieve contextual information for your tab content, see [get context for your Teams tab](../tabs/how-to/access-teams-context.md#get-context-by-using-the-microsoft-teams-javascript-library). `meetingId` is used by a tab when running in the meeting context and is added for the response payload.
+To identify and retrieve contextual information for your tab content, see [get context for your Teams tab](../tabs/how-to/access-teams-context.md#get-context-by-using-the-microsoft-teams-javascript-library). `meetingId` is used by a tab running in the meeting context and is added for the response payload.
 
-## GetParticipant API
+## Get participant API
 
 > [!NOTE]
 > * Do not cache participant roles since the meeting organizer can change the roles any time.
-> * Teams does not currently support large distribution lists or roster sizes of more than 350 participants for the `GetParticipant` API.
-
-The `GetParticipant` API allows a bot to fetch participant information by meeting ID and participant ID. The API includes query parameters, examples, and response codes.
+> * Currently, limitation to distribution lists or roster sizes are less than 350 participants for the `GetParticipant` API.
```

Comment:
Currently, this API is only supported for distributions lists or rosters with less than 350 participants.